interface/main/messages/save.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * /interface/main/messages/save.php
+ *
+ * Copyright (C) 2017 MedEx <support@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package OpenEMR
+ * @author MedEx <support@MedExBank.com>
+ * @link http://www.open-emr.org
+ */
+
+
+require_once "../../globals.php";
+require_once "$srcdir/acl.inc";
+require_once "$srcdir/lists.inc";
+require_once "$srcdir/api.inc";
+require_once "$srcdir/forms.inc";
+require_once "$srcdir/patient.inc";
+require_once "$srcdir/MedEx/API.php";
```

Comment:
See my comment above. Du need to use `use` statements, just keep it high level or use aliases